src/Commands/MakeRouteCommand.php

```diff
@@ -16,7 +16,7 @@ public function handle()
 
         beyond_copy_stub(
             'routes.stub',
-            app_path() . "/../src/App/{$application}/routes.php",
+            app_path() . "/../src/App/{$application}/routes/web.php",
```

Comment:
This would overwirte the `web.php` file of Laravel. Take the application name as file name (lowercase)